spec/setup_acceptance_node.pp

```diff
@@ -0,0 +1,37 @@
+class { 'foreman::repo':
+  repo => 'nightly',
+}
+class { 'katello::repo':
+  repo_version => 'nightly',
+}
+class { 'candlepin::repo':
+  version => 'nightly',
+}
+
+user { 'vagrant':
+  ensure     => present,
+  managehome => true,
+}
+
+# Necessary for PostgreSQL EVR extension
+yumrepo { "pulpcore":
+  baseurl  => "http://yum.theforeman.org/pulpcore/3.14/el\$releasever/x86_64/",
+  descr    => "Pulpcore",
+  enabled  => true,
+  gpgcheck => true,
+  gpgkey   => "https://yum.theforeman.org/pulpcore/3.14/GPG-RPM-KEY-pulpcore",
+}
```

Comment:
Not just for EVR, also Pulp itself. However, we should have the Pulpcore module here
```suggestion
class { 'pulpcore::repo':
}
```

Reply:
We should, if this module actually did anything with Pulpcore :) This is similar to the puppet-katello acceptance tests, this does not rely on puppet-foreman_proxy_content which handles setting up Pulp 3. And for the installer, the answer file brings the modules together. This module has no knowledge of getting Pulp set up.

Reply:
Ah right